components/avatar/group.tsx

```diff
@@ -26,7 +27,14 @@ export interface GroupProps {
 
 const Group: React.FC<GroupProps> = props => {
   const { getPrefixCls, direction } = React.useContext(ConfigContext);
-  const { prefixCls: customizePrefixCls, className = '', maxCount, maxStyle, size } = props;
+  const {
+    prefixCls: customizePrefixCls,
+    className = '',
+    maxCount,
+    maxStyle,
+    shape = 'circle',
```

Comment:
```suggestion
   maxShape = 'circle',
```

My bad... `maxShape` is better here, I misunderstood the meaning.

Reply:
@MadCcc What should I do? Do you want me to push a commit to rename `shape` back to `maxShape`?

Reply:
Just a follow up @MadCcc. Do you want me to push a commit to rename `shape` back to `maxShape`?